Makefile

```diff
@@ -74,31 +50,20 @@ all: format build clippy test doc e2e
 build:
 	$(CARGO) build --workspace --all-targets $(BUILD_ARGS)
 
+lint: clippy ts_lint
+
+ts_lint:
+	(cd ./tests; yarn install; yarn check)
+
 clippy: install_clippy
 	$(CARGO) clippy --all-targets -- -D warnings
 
 e2e:
 	$(CARGO) build -p cnd -p nectar $(BUILD_ARGS)
 	(cd ./tests; yarn install; yarn test)
 
-check_format: check_rust_format check_toml_format check_ts_format
-
-format: install_rustfmt install_tomlfmt
-	$(CARGO_NIGHTLY) fmt -- --files-with-diff | xargs -I{} git add {}
-	@$(foreach file,$(CARGO_TOML_FILES),$(CARGO) tomlfmt -p $(file) && git add $(file);)
-ifneq (,$(MODIFIED_TYPESCRIPT_FILES))
-	(cd ./tests; yarn install; yarn run fix)
-endif
+check_format:
+	dprint check
```

Comment:
How do I install dprint?

Reply:
Which ever way you prefer: https://dprint.dev/install/

I've changed my mind on this auto-installing tools topic. What is important is that versions of tools are enforced, like we do for `node` for example but devs should be able to decide themselves, how they want to manage their environment.

Reply:
In the case of dprint, what is important are the versions + configuration of the plugins. Both are in the dprint configuration file so the actual version of `dprint` is not that important!